dsaps/helpers.py

```diff
@@ -48,21 +42,21 @@ def create_metadata_id_list(metadata_csv):
     return metadata_ids
 
 
-def match_files_to_metadata(file_dict, metadata_ids):
+def match_files_to_metadata(file_list, metadata_ids):
     """Creates a list of files matched to metadata records."""
     file_matches = []
-    for file_id, v in file_dict.items():
+    for file_id in file_list:
         for metadata_id in [m for m in metadata_ids
                             if file_id.startswith(m)]:
             file_matches.append(file_id)
     return file_matches
 
 
-def match_metadata_to_files(file_dict, metadata_ids):
+def match_metadata_to_files(file_list, metadata_ids):
     """Creates a list of metadata records matched to files."""
     metadata_matches = []
```

Comment:
This can be similarly simplified with a single list comprehension: `metadata_matches = [metadata_id for f in file_list for metadata_id in metadata_ids if f.startswith(metadata_id)]`. Also, the test for this function needs to be updated to use a list instead of a dict.